src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +35,11 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return getModifiedNumbers(numbers).stream()
+                .filter(isOddNumber -> isOddNumber % 2 != 0)
```

Comment:
create an isOddNumber() method and reuse it for other streams
```suggestion
                .filter(num -> isOddNumber(num))
```